lib/ProxyManager/ProxyForm.js

```diff
@@ -82,7 +93,7 @@ class ProxyForm extends React.Component {
     ));
 
     return (
-      <form id="proxy-form" onSubmit={handleSubmit}>
+      <form id="proxy-form" onSubmit={e => this.handleSubmitLocal(e)}>
```

Comment:
```suggestion
      <form id="proxy-form" onSubmit={this.handleSubmitLocal}>
```
It seems to work too.